backend/src/Designer/Services/Implementation/Organisation/OrgTextsService.cs

```diff
@@ -88,6 +89,19 @@ public async Task UpdateTextsForKeys(string org, string developer, Dictionary<st
         await altinnOrgGitRepository.SaveText(languageCode, textResourceObject, cancellationToken);
     }
 
+    private static async Task EnsureTextResourceExists(AltinnOrgGitRepository altinnOrgGitRepository, string languageCode, CancellationToken cancellationToken)
+    {
+        try
+        {
+            await altinnOrgGitRepository.GetText(languageCode, cancellationToken);
+        }
+        catch (NotFoundException)
+        {
+            var emptyTextResource = new TextResource { Language = languageCode, Resources = [] };
+            await altinnOrgGitRepository.SaveText(languageCode, emptyTextResource, cancellationToken);
```

Comment:
Vi trenger ikke å hente innholdet i tekst ressurs filen her, kanskje en ny funksjon i `AltinnOrgGitRepository` kan være bedre?

Her er et forslag til hvordan funksjonen kan se ut:
```cs
    public static bool ResourceFileExists(string languageCode)
    {
        string fileName = GetTextResourceFileName(languageCode);
        return File.Exists(GetPathToJsonTextsFile(fileName));
    }
```

Og eventuelt dra ut fil navnet i en egen funksjon
```cs
    private static string GetTextResourceFileName(string languageCode)
    {
        return $"resource.{languageCode}.json";
    }
```

Da vil vi kunne gjøre det slik her:
```suggestion
        if (!AltinnOrgGitRepository.ResourceFileExists(languageCode))
        {
            var emptyTextResource = new TextResource { Language = languageCode, Resources = [] };
            await altinnOrgGitRepository.SaveText(languageCode, emptyTextResource, cancellationToken);
        }
```

Reply:
Veldig godt tips!
Jeg så også at andre metoder i `AltinnOrgGitRepository` kunne dra nytte av denne, så jeg tilpasset forslaget ditt litt i siste commit 🙂